src/app.js

```diff
@@ -1,21 +1,7 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const { moveFile } = require('./moveFile');
```

Comment:
You are using process.argv without any validation. It would be better to handle the case where the user does not provide the expected arguments to avoid potential errors.